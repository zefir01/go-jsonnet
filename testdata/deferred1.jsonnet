local a={
    _schema::{
        f4: ['aaa', {vv: 1}]
    }
};

local b= {
  t0: a.f4[1],
  t4: a.f4,
  t4_d: std.isDeferred(a.f4),
  t5: a.f4[0],
  t5_d: std.isDeferred(a.f4[0]),
};

{
  t1: b,
  s: std.isDeferred(b),
}