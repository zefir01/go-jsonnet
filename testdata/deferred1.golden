{
   "s": true,
   "t1": {
      "t0": {
         "vv": 1
      },
      "t4": [
         "aaa",
         {
            "vv": 1
         }
      ],
      "t4_d": true,
      "t5": "aaa",
      "t5_d": true
   }
}
